system-addon/jar.mn

```diff
@@ -17,13 +17,28 @@
   content/vendor/react-intl.js (./vendor/react-intl.js)
   content/vendor/redux.js (./vendor/redux.js)
   content/vendor/react-redux.js (./vendor/react-redux.js)
-  content/data/ (./data/*)
+  content/data/content/locales.json (./data/locales.json)
+  content/data/content/assets/ (./data/content/assets/*)
+  content/data/content/tippytop/ (./data/content/tippytop/*)
+  content/data/content/activity-stream.bundle.js (./data/content/activity-stream.bundle.js)
+#ifndef RELEASE_OR_BETA
+  content/data/content/activity-stream.bundle.js.map (./data/content/activity-stream.bundle.js.map)
+#endif
 #ifdef XP_MACOSX
   content/css/activity-stream.css (./css/activity-stream-mac.css)
+  #ifndef RELEASE_OR_BETA
```

Comment:
Not sure if indenting actually works?  At least I don't see other places indenting…

Reply:
Based on manual testing, it looks like this does work

Reply:
Hm, nevermind. The inner block seems to get ignored